snuba/datasets/schemas/__init__.py

```diff
@@ -124,18 +123,10 @@ def get_column_differences(
             if isinstance(expected_type, ReadOnly):
                 expected_type = expected_type.inner_type
 
-            # TODO: Remove if statement once modifiers are only used in migrations
-            if column != expected_type:
-                while True:
-                    if isinstance(column, ColumnTypeWithModifier):
-                        column = column.inner_type
-                    else:
-                        break
-
-                if column != expected_type:
-                    errors.append(
-                        "Column '%s' type differs between local ClickHouse and schema! (expected: %s, is: %s)"
-                        % (column_name, expected_type, column)
-                    )
+            if column.get_raw() != expected_type.get_raw():
```

Comment:
Well, we still compare that the raw type matches (which now includes the nullability component too). We no longer compare that the modifiers - i.e. the materialized, withdefault, lowcardinality, and codec parts match - but that's because we no longer duplicate this information so I think comparing that is of little value. I think the main reason for this existing previously was to ensure the old and new migration systems were kept in sync and that is no longer required.